README.md

```diff
@@ -113,5 +113,45 @@ Setting `FALLBACK=https://scratch.mit.edu` allows the web client to retrieve dat
 
 Additionally, if you set `FALLBACK=https://scratch.mit.edu`, be aware that clicking on links to parts of the website not yet migrated over (currently such as `Explore`, `Discuss`, `Profile`, etc.) will take you to the Scratch website itself.
 
+#### Docker
+
+A set of [Docker](https://www.docker.com/what-docker) related files are provided to create isolated [container](https://www.docker.com/what-container) environments suitable for development:
+
+* Dockerfile
+* docker-compose.yml
+* docker_entrypoint.sh
+
+Unless you have access to the Scratch REST API repository, this may not be very useful for you.
+
+This currently requires a version of the [Scratch GUI](https://github.com/LLK/scratch-gui) repository to be checked out in parallel to scratch-www, e.g.:
```

Comment:
This is hopefully very temporary. Once the export scratch-gui for the projects page is in the published version of gui, you won't **need** the parallel gui.

Reply:
Great!